templates/governance/Releasefile.yaml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
this test is failing due to existing issue in xl-cli ,
https://www7.v1host.com/V1Production/story.mvc/Summary?oidToken=Story%3a2609895

it has to be fixed to verify this apply